src/main/java/org/opentripplanner/graph_builder/linking/StreetSplitter.java

```diff
@@ -1,11 +1,834 @@
+            } else {
+                List<TransitStop> bestStops = Lists.newArrayList();
+                // Add stops until there is a break of epsilon meters.
+                // we do this to enforce determinism. if there are a lot of stops that are all extremely close to each other,
+                // we want to be sure that we deterministically link to the same ones every time. Any hard cutoff means things can
+                // fall just inside or beyond the cutoff depending on floating-point operations.
+                int i = 0;
+                do {
+                    bestStops.add(candidateStops.get(i++));
+                } while (i < candidateStops.size() &&
+                    stopDistances.get(candidateStops.get(i).getIndex()) - stopDistances
+                        .get(candidateStops.get(i - 1).getIndex()) < DUPLICATE_WAY_EPSILON_DEGREES);
+
+                for (TransitStop stop: bestStops) {
+                    LOG.debug("Linking vertex to stop: {}", stop.getName());
+                    makeTemporaryEdges((TemporaryStreetLocation)vertex, stop, destructiveSplitting);
+                }
+                return true;
+            }
+        } else {
+
+            // find the best edges
+            List<StreetEdge> bestEdges = Lists.newArrayList();
+
+            // add edges until there is a break of epsilon meters.
+            // we do this to enforce determinism. if there are a lot of edges that are all extremely close to each other,
+            // we want to be sure that we deterministically link to the same ones every time. Any hard cutoff means things can
+            // fall just inside or beyond the cutoff depending on floating-point operations.
+            int i = 0;
+            do {
+                bestEdges.add(candidateEdges.get(i++));
+            } while (i < candidateEdges.size() &&
+                distances.get(candidateEdges.get(i).getId()) - distances
+                    .get(candidateEdges.get(i - 1).getId()) < DUPLICATE_WAY_EPSILON_DEGREES);
+
+            for (StreetEdge edge : bestEdges) {
+                linkToEdge(vertex, edge, xscale, options, destructiveSplitting, createSemiPermanentEdges);
+            }
+
+            // Warn if a linkage was made, but the linkage was suspiciously long.
+            if (vertex instanceof TransitStop) {
+                double distanceDegreesLatitude = distances.get(candidateEdges.get(0).getId());
+                int distanceMeters = (int)SphericalDistanceLibrary.degreesLatitudeToMeters(distanceDegreesLatitude);
+                if (distanceMeters > WARNING_DISTANCE_METERS) {
+                    // Registering an annotation but not logging because tests produce thousands of these warnings.
+                    graph.addBuilderAnnotation(new StopLinkedTooFar((TransitStop)vertex, distanceMeters));
+                }
+            }
+
+            return true;
+        }
+    }
+
+    // Link to all vertices in area/platform
```

Comment:
javadoc format needed

Reply:
I'll add in the javadoc format, but I don't exactly know much about the original code. The reason it's showing up green is that the SimpleStreetSplitter file is showing up as deleted when it's really just taken over this StreetSplitter file.